inductiva/fluids/scenarios/wind_tunnel/wind_tunnel.py

```diff
@@ -50,9 +68,9 @@ def __init__(self,
         """Initializes the `WindTunnel` conditions.
 
         Args:
-            flow_velocity (dict): Velocity of the air flow in m/s.
+            flow_velocity (dict): Velocity of the air flow (m/s).
             domain (dict): List containing the lower and upper boundary of
-                the wind tunnel in each (x, y, z) direction. It is the
+                the wind tunnel in each (x, y, z) direction (m). It is the
                 natural description with the default OpenFOAM simulator.
```

Comment:
Can we make this simpler, and take only a list as input, `tunnel_size`, e.g. `tunnel_size=[10, 3, 3]`. You would then build whatever description OpenFOAM consumes in the scenario.

Reply:
This was input previously given by Sarmento.

Reply:
@sarmento WDYT?